content/en/database_monitoring/guide/connect_dbm_and_apm.md

```diff
@@ -353,26 +353,35 @@ Enable the database monitoring propagation feature by setting the following envi
 
 {{< /tabs >}}
 
-## Explore the APM Connection
+## Explore the APM Connection in DBM
 
 ### Attribute active database connections to the calling APM services
 
 {{< img src="database_monitoring/dbm_apm_active_connections_breakdown.png" alt="View active connections to a database broken down by the APM Service they originate from.">}}
 
 Break down active connections for a given host by the upstream APM services making the requests. You can attribute load on a database to individual services to understand which services are most active on the database. Pivot to the most active upstream service’s service page to continue the investigation.
 
+### Filter the Databases List for specific APM calling services.
```

Comment:
can this get a short blurb that just says like "filter your database hosts according to what services call them" but... nicer?

Reply:
Updated to friendly wording